tests/test_record_resource.py

```diff
@@ -35,7 +35,6 @@ def test_create_read_record(client, input_record):
 
```

Comment:
Moderate (Applies to the whole file):

Align with @fenekku on this:

- There is a refactor
- @fenekku 's PR uses `/records` (I would say this is the way to go)
- This PR has a different way of handling the skip (IMO it's better since tests will still run on travis, but we avoid the hassle in local).

Reply:
Yes I saw that :/ If we merge his PR first I will do the changes :) It's just that currently I need to keep it like that! But, I totally agree with the changes he made!

Reply:
I will also need to apply the error handling the other views he created most probably, so it might be better to merge his first :)